.browserslistrc

```diff
@@ -0,0 +1,4 @@
+last 2 Chrome Major versions
```

Comment:
Uncertain

Reply:
Is this just for linting or does it affect output produced by babel/webpack? We definitely want/need to support more than the latest 2 versions of Chrome. I don't know how frequently schools update software but we need to consider the slowest updaters here, not just the average.

It is also a bit tricky that different parts of the app have different requirements (FF52 and Edge 18 is only required for the student part, not teacher features). 

Are there any warnings on the current codebase with these settings?

Reply:
> Is this just for linting or does it affect output produced by babel/webpack? We definitely want/need to support more than the latest 2 versions of Chrome. I don't know how frequently schools update software but we need to consider the slowest updaters here, not just the average.

It's just for linting. 
Looking at exam documentation it states.
Chrome: All fairly recent versions, which can be interpreted pretty freely.
Perhaps it would be better with a time based query here (last x years), as the interval between major releases differs.

> It is also a bit tricky that different parts of the app have different requirements (FF52 and Edge 18 is only required for the student part, not teacher features).

I don't see an immediate solution to adopt the linting this way. Perhaps it's enough if it's changed from errors to warnings. 

> Are there any warnings on the current codebase with these settings?

![Screenshot from 2022-04-01 08-14-18](https://user-images.githubusercontent.com/30393363/161207904-6f385559-3d65-4c5d-b855-bd6786cf55ee.png)

Reply:
Not too bad with the errors, perhaps we could/should add eslint-ignore lines for these, if they are intentional. Or wrap the code in conditions if that is understood by this linting.

The `Audio.vue`-warning is a bit strange, `body` is not called as a method in the code.

For Chrome versions, I agree, perhaps something like "last 5 years" is the best. There is however a big difference between using something that crashes in older browsers, or using something which just gives a new feature/convenience in newer browsers. Ideally, we could recommend schools to keep browsers updated to get all latest features, but having 99 % of the system still work even if they are a few years old.

Reply:
Seems to ignore if inside if. 

Yeah a slightly misguiding. I think the error is supposed to be window.body not window.body()

Reply:
Ah, I thought it was a pure bug since `document.body` has been around since forever. It seems to always have existed in HTML but not for other kinds of documents prior to FF60
<img width="608" alt="image" src="https://user-images.githubusercontent.com/25909128/161218601-905a8aef-b9ab-4152-94d5-a771bc71ccac.png">